dcos/http.py

```diff
@@ -162,11 +174,23 @@ def request(method,
     if 'headers' not in kwargs:
         kwargs['headers'] = {'Accept': 'application/json'}
 
-    response = _request(method, url, is_success, timeout, **kwargs)
+    if verify is None and os.environ.get(constants.DCOS_SSL_VERIFY_ENV):
```

Comment:
I think you want: `if verify is None and constants.DCOS_SSL_VERIFY_ENV in os.environ:`